experiments.py

```diff
@@ -68,9 +68,9 @@ def save_results_dict(arguments, results_dict, f_name_prefix = ""):
             rc_dir_str = "results/{}/{}/{}/{}".format(arguments["experiment name"],
                 arguments["dataset name"], arguments["base model"]["name"],recourse_name).replace(" ", "")
             pathlib.Path(rc_dir_str).mkdir(parents=True, exist_ok=True) 
-            recourse_results[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'trial_results.csv', index=False)
-            agg_recourse_results[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'agg.csv', index=False)
-            agg_recourse_results_err[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'agg_err.csv', index=False)
+            recourse_results[recourse_name].to_csv(rc_dir_str+'/'+f_name_prefix+'all_trials_results.csv', index=False)
```

Comment:
`os.path.join()` might be your friend instead of hardcoding in directory slashes. pathlib might have an equivalent.